tests/routes/post_offerer_test.py

```diff
@@ -96,3 +99,82 @@ def expect_the_current_user_to_be_editor_of_the_new_offerer(self, app):
             assert response.status_code == 201
             offerer = Offerer.query.first()
             assert offerer.UserOfferers[0].rights == RightsType.editor
+
+
+        @clean_database
+        def when_offerer_is_new_he_has_validation_token_and_user_offerer_dont(self, app):
```

Comment:
Je suis un peu (vraiment un peu), gêné par le "he", je ne sais pas si on ne pourrais pas s'en défaire.

```suggestion
        def a_new_offerer_has_a_validation_token_but_user_offerer_dont(self, app):
```